package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -73,21 +65,33 @@ def search(self, atoms, radius, level='A'):
 
         Parameters
         ----------
-        atoms : AtomGroup, MDAnalysis.core.groups.Atom
-          list of atoms
+        atoms : AtomGroup, MDAnalysis.core.groups.AtomGroup
+          AtomGroup object
         radius : float
           Radius for search in Angstrom.
         level : str
           char (A, R, S). Return atoms(A), residues(R) or segments(S) within
           *radius* of *atoms*.
+
+        .. versionchanged:: 2.0.0
+
+        Returns
+        -------
+        AtomGroup : :class:`MDAnalysis.core.groups.AtomGroup`
+          When ``level='A'``, AtomGroup is being returned.
+        ResidueGroup : :class:`MDAnalysis.core.groups.ResidueGroup`
+          When ``level='R'``, ResidueGroup is being returned.
+        SegmentGroup : :class:`MDAnalysis.core.groups.SegmentGroup`
+          When ``level='S'``, SegmentGroup is being returned.
```

Comment:
The versionchanged typically comes at the end or for specific parameters, under the parameter. In this case, just do
```suggestion

        Returns
        -------
        AtomGroup : :class:`~MDAnalysis.core.groups.AtomGroup`
          When ``level='A'``, AtomGroup is being returned.
        ResidueGroup : :class:`~MDAnalysis.core.groups.ResidueGroup`
          When ``level='R'``, ResidueGroup is being returned.
        SegmentGroup : :class:`~MDAnalysis.core.groups.SegmentGroup`
          When ``level='S'``, SegmentGroup is being returned.
          
          
        .. versionchanged:: 2.0.0
           Now returns :class:`AtomGroup` (when empty this is now an empty
           :class:`AtomGroup` instead of an empty list), :class:`ResidueGroup`,
           or a :class:`SegmentGroup`
```

Reply:
Thank you for the suggestion. I have clicked the commit suggestion, which should just apply the change.